app/lib/get_started_form.rb

```diff
@@ -70,4 +76,8 @@ def plan_term_s
   def technical_area_ids
     plan_by_technical_ids? ? @technical_area_ids : []
   end
+
+  def valid_diseases?
+    errors.add(:diseases, "Invalid disease id") unless diseases.nil? || diseases.inject(true) { |included, disease| included && Plan::DISEASE_TYPES.include?(disease) }
```

Comment:
Could we instead use `diseases.all? {...}`

Reply:
fixed